force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +78,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if (b == 0) {
```

Comment:
Nice! Its also a good idea to add a null parameter check at the beginning of your methods.